crates/ide/src/syntax_highlighting/highlight.rs

```diff
@@ -255,6 +255,7 @@ pub(super) fn element(
                     })
                     .map(|modifier| h | modifier)
                     .unwrap_or(h),
+                T![async] | T![await] => h | HlMod::Async,
```

Comment:
`T![await]` is being matched in an earlier arm already as its being tagged with ControlFlow as well, you'll have to special case that one

Reply:
Modified, now only match the async.

Reply:
Ah sorry I meant removing `T![await]` from the other arms and adding `T![await] => h | HlMod::Async | HlMod::ControlFlow`, as its both control flow and async.

Reply:
Sorry, I misunderstood you.

Reply:
Fixed.

Reply:
Thanks for your help!

Reply:
No problem